src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_parser.py

```diff
@@ -99,7 +99,7 @@ def _get_model(self, data: Data, file_name: str) -> Model:
                     )
                     for well in data.wells
                 ],
-                calculated_data_aggregate_document=self.get_outter_calculated_data_aggregate_document(
+                calculated_data_aggregate_document=self.get_outer_calculated_data_aggregate_document(
```

Comment:
nice, but could we move this rename out of this fixing PR?

Reply:
No